scrapper/databases/sqlite_operations.py

```diff
@@ -0,0 +1,10 @@
+def create_db_if_not_exists(pages_to_scrap):
+    pass
+
+
+def insert_record(flat_dictionary):
+    pass
+
+
+def check_if_link_exists(url):
+    pass
```

Comment:
Tak jak mówiłem, logiczna kolejność wg mnie to najpierw sprawdzanie, czy link istnieje a potem dodawanie do bazy. Nawet jeśli nie ma tu kolejność żadnego znaczenia w tym miejscu, to myślę, że warto to tak pisać.

Reply:
fixed